state/store.go

```diff
@@ -211,7 +211,7 @@ func LoadValidators(db dbm.DB, height int64) (*types.ValidatorSet, error) {
 				),
 			)
 		}
-		valInfo2.ValidatorSet.IncrementProposerPriority(tmmath.SafeConvertInt32(height - lastStoredHeight)) // mutate
+		valInfo2.ValidatorSet.IncrementProposerPriority(tmmath.SafeConvertInt32(int(height - lastStoredHeight))) // mutate
```

Comment:
`int(height-lastStoredHeight)` is not safe on 32 bit arch

Reply:
I guess it's fine since we don't support 32-bit anyway

Reply:
since we are only using this here, it would be safer to have the fun accept a int64. will make the change then merge